public/assets/images/empty_color.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 17.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
Run SVG's thru https://jakearchibald.github.io/svgomg/ to minimize byte size. It will remove anything unneeded, like the Adobe credit.